blockly-core/core/connections/connection.js

```diff
@@ -616,9 +665,18 @@ Blockly.Connection.prototype.setCheck = function(check) {
     if (!(check instanceof Array)) {
       check = [check];
     }
+
     this.check_ = check;
+
+    var legacyTypeFound = Blockly.Connection.findLegacyType_(check);
+    if (legacyTypeFound) {
+      console.log("Legacy Type Found:");
+      console.log(legacyTypeFound);
+      throw 'Error: found legacy type check (see console)';
```

Comment:
any reason not to just put the type in the error message we throw? I think we've managed to mostly keep console.log statements out of our code